meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -65,10 +64,6 @@ FILES:${PN}-lite = " \
                     "
 FILES:${PN}-lite-lib = "${nonarch_libdir}/lib${PN}_lite.so"
 FILES:${PN}-lite-dev = "${includedir}/${PN}-lite"
-FILES:${PN}-dev = " \
```

Comment:
I suppose it's needed for a custom client.

Reply:
Same here.